src/cluster/backend/zeromq/ZeroMQ-Proxy.h

```diff
@@ -54,6 +55,7 @@ class ProxyThread {
     Args args;
     std::string xpub_endpoint;
     std::string xsub_endpoint;
+    int ipv6 = 1;
```

Comment:
Is there a reason these flags aren't booleans?

Reply:
The ZeroMQ API takes `int` values, so this seemed appropriate here.